types/types.go

```diff
@@ -202,8 +202,40 @@ type (
 		SenderRewards   MicroAlgos `codec:"rs"`
 		ReceiverRewards MicroAlgos `codec:"rr"`
 		CloseRewards    MicroAlgos `codec:"rc"`
+		EvalDelta       EvalDelta  `codec:"dt"`
 	}
 
+	// EvalDelta stores StateDeltas for an application's global key/value store, as
+	// well as StateDeltas for some number of accounts holding local state for that
+	// application
+	EvalDelta struct {
```

Comment:
I just did a little digging and see that this is part of ApplyData, the delta portion makes a lot more sense now.